suite-native/staking/src/cardanoStakingSelectors.ts

```diff
@@ -0,0 +1,73 @@
+import { createWeakMapSelector } from '@suite-common/redux-utils';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import {
+    AccountsRootState,
+    selectAccountByKey,
+    selectDeviceAccounts,
+} from '@suite-common/wallet-core';
+import {
+    getStakingDataForNetwork,
+    isCardanoStakedWithFiveBinaries,
+} from '@suite-common/wallet-utils';
+
+import { NativeStakingRootState } from './types';
+
+export const createMemoizedSelector = createWeakMapSelector.withTypes<NativeStakingRootState>();
+
+export const selectVisibleDeviceCardanoAccountsWithStakingByNetworkSymbol = createMemoizedSelector(
+    [selectDeviceAccounts, (_state, symbol: NetworkSymbol) => symbol],
+    (accounts, symbol) =>
+        accounts.filter(
+            account =>
+                account.symbol === symbol &&
+                account.visible &&
+                account.networkType === 'cardano' &&
+                account.misc?.staking?.isActive,
```

Comment:
```
            account =>
                account.symbol === symbol &&
                account.visible &&
                account.networkType === 'cardano' &&
                account.misc?.staking?.isActive,
```
I have already seen this condition at least three times across this PR. Is it possible to extract it into a separate selector or util that will be used everywhere? WDYT?